portfolio/src/main/webapp/script.js

```diff
@@ -87,3 +87,33 @@ async function getRandomQuoteUsingAsyncAwait() {
   document.getElementById('quote-container').innerText = quote;
 }
 
+/**
+ * Fetches quotess from the servers and adds them to the DOM.
+ */
+function getQuote() {
```

Comment:
Looks like "getQuote" is a pretty generic name here, considering there are other functions that also "get quote" in this class. Also it seems to me from the functionality that it's getting all quotes - so maybe rename to getQuotesFromServer?
I think what's missing here is you have filled out this method, but haven't actually used/called it anywhere - think about where you can call this function to display the desired output on your page